jupyter_server/serverapp.py

```diff
@@ -1214,6 +1215,21 @@ def _deprecated_password_config(self, change: t.Any) -> None:
         """,
     )
 
+    allow_unauthenticated_access = Bool(
```

Comment:
For ease of use in JupyterHub deployments, I would also suggest making this available as an env variable. I know you can setup a _config.json file reasonably easily, but given the possible high impact here, I think it's worth making an env var for this too. `JUPYTER_SERVER_ALLOW_UNAUTHENTICATED_ACCESS` or similar?

Reply:
Done in [cd84175](https://github.com/jupyter-server/jupyter_server/pull/1392/commits/cd84175921f2d1d3008b3184ca1caf22e18ea17d)